TimerCalculatorUtility/TimerCalculatorUtility/Enums/StopwatchStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace TimerCalculatorUtility.Enums
+{
+    public enum StopwatchStatus
+    {
+        Onstart,
```

Comment:
Use Pascal Case. Also, Onstart doesn't show state, it would be better to name it Started instead of Onstart.

Reply:
> Use Pascal Case. Also, Onstart doesn't show state, it would be better to name it Started instead of Onstart.

This status means condition of app before click on "Start" button, but "Started" is like stopwatch arleady running first lap. For this situation i have status "Processing" yet.